src/measure_theory/constructions/borel_space.lean

```diff
@@ -170,6 +170,60 @@ the `σ`-algebra of measurable sets is exactly the `σ`-algebra generated by ope
 class borel_space (α : Type*) [topological_space α] [measurable_space α] : Prop :=
 (measurable_eq : ‹measurable_space α› = borel α)
 
+namespace tactic
+
+/-- Add instances `borel α : measurable_space α` and `⟨rfl⟩ : borel_space α`. -/
+meta def add_borel_instance (α : expr) : tactic unit :=
+do
+  n1 ← get_unused_name "_inst",
+  to_expr ``(borel %%α) >>= pose n1,
+  reset_instance_cache,
+  n2 ← get_unused_name "_inst",
+  t ← to_expr ``(borel_space %%α),
+  v ← to_expr ``(borel_space.mk rfl),
+  note n2 t v,
```

Comment:
This looks a little suspicious. `v` is getting elaborated without an expected type. Maybe better:
```suggestion
  v ← to_expr ``(borel_space.mk rfl : borel_space %%α),
  note n2 none v
```